package.yaml

```diff
@@ -0,0 +1,68 @@
+name: interpolator
+version: '1.0'
+maintainer: TVision Insights
+license: AllRightsReserved
```

Comment:
Actually, _here_ you need a `0.x` version and a license. Not such a big deal in the example package I guess.

Reply:
Seems like packages in Hackage always have 3 or 4 segments, so 0.1.0 or 0.1.0.0? I'm not gonna sweat it though, as long as it starts with `0.`.